packages/elements/src/photon-multirx-form/index.tsx

```diff
@@ -331,14 +328,7 @@ customElement(
                 <sl-spinner style="font-size: 3rem;"></sl-spinner>
               </div>
             </Show>
-            <Show when={client && !authenticated() && hasError()}>
```

Comment:
This message was incorrect before (not being authenticated != not being authorized to prescribe), but i think we should show some error in this case